content/doc/book/managing/plugins.adoc

```diff
@@ -225,6 +225,67 @@ The configuration(s) created by the disabled plugin behave as if the plugin
 were uninstalled, insofar that they result in warnings on boot but are
 otherwise ignored.
 
+==== Using the Jenkins CLI
+
+Making use of the <<cli#,Jenkins CLI>> it is also possible to enable and disable
+plugins. Since Jenkins 2.136, administrators may invoke the `enable-plugin` 
+command to re-enable again a collection of plugins which were previously disabled
+and since Jenkins 2.151, the `disable-plugin` command is included, which provides
+a command to disable installed plugins.
+
+The `enable-plugin` command receives as parameter a list of plugins to be enabled.
+If any of the plugins have one or more dependencies, these plugins will be enabled
+as well.
+
+[source]
+----
+java -jar jenkins-cli.jar -s http://localhost:8080/ enable-plugin PLUGIN ... [-restart]
+
+Enables one or more installed plugins transitively.
+
+ PLUGIN   : Enables the plugins with the given short names and their
+            dependencies.
+ -restart : Restart Jenkins after enabling plugins.
+----
+
+The `disable-plugin` command receives as parameter a list of plugins to disable.
+The output will display messages for both successful and failed operations. In 
+case that only the error messages are desired, the execution should add the 
+`-quiet` option. How to deal with other plugins which depend on one or more of 
+the plugins passed as parameter may be specified with the `-strategy` option.
+
+[source]
+----
+java -jar jenkins-cli.jar -s http://localhost:8080/ disable-plugin PLUGIN ... [-quiet (-q)]
+[-restart (-r)] [-strategy (-s) strategy]
+
+Disable one or more installed plugins.
+Disable the plugins with the given short names. You can define how to proceed with the 
+dependant plugins and if a restart after should be done. You can also set the quiet mode 
+to avoid extra info in the console.
+
+ PLUGIN                  : Plugins to be disabled.
+ -quiet (-q)             : Be quiet, print only the error messages
+ -restart (-r)           : Restart Jenkins after disabling plugins.
+ -strategy (-s) strategy : How to process the dependant plugins. 
+                           - none: if a mandatory dependant plugin exists and
+                           it is enabled, the plugin cannot be disabled
+                           (default value).
+                           - mandatory: all mandatory dependant plugins are
+                           also disabled, optional dependant plugins remain
+                           enabled.
+                           - all: all dependant plugins are also disabled, no
+                           matter if its dependency is optional or mandatory.
+----
+
+[CAUTION]
+====
+In the same way than enabling and disabling plugins from the UI need a restart 
+to complete the process, the changes made with the CLI commands will take effect
+once Jenkins is restarted. The `-restart` option forces the instance to restart
+once the command has successfully finished, so the changes will be immediately
+applied.
```

Comment:
Unclear which kind of restart (safe or otherwise).

Reply:
@MRamonLeon @jvz Could you clarify @daniel-beck's doubt?

Reply:
Specifically: ```Jenkins.get().safeRestart();```

Reply:
Thank @MRamonLeon !